crypto/rsa/rsa_lib.c

```diff
@@ -744,9 +744,9 @@ int RSA_pkey_ctx_ctrl(EVP_PKEY_CTX *ctx, int optype, int cmd, int p1, void *p2)
 
 DEFINE_STACK_OF(BIGNUM)
 
-int ossl_rsa_set0_all_params(RSA *r, const STACK_OF(BIGNUM) *primes,
-                             const STACK_OF(BIGNUM) *exps,
-                             const STACK_OF(BIGNUM) *coeffs)
+int ossl_rsa_set0_all_params(RSA *r, STACK_OF(BIGNUM) *primes,
```

Comment:
Maybe comment at the top here that this function removes elements from the passed in stacks (since this is a bit unusual).

Reply:
done